packages/presentation/src/image.ts

```diff
@@ -0,0 +1,96 @@
+//
+// Copyright © 2024 Hardcore Engineering Inc.
+//
+// Licensed under the Eclipse Public License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may
+// obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import extract from 'png-chunks-extract'
+
+export async function getImageSize (file: File, src: string): Promise<{ width: number, height: number }> {
+  const size = isPng(file) ? await getPngImageSize(file) : undefined
```

Comment:
This is not correct, you need to have 3 values, width, height and dpi or pixelRatio. So we could decide later how to deal with them.

Reply:
Added